packages/pages/src/components/map/marker.tsx

```diff
@@ -65,31 +43,41 @@ export const Marker = ({
 
     return () => {
       marker.setMap(null);
-    }
+    };
   }, []);
 
-  if (children) {
+  const elementToRender = children ? children : icon;
+
+  if (elementToRender) {
     const pinEl = marker.getProviderPin().getPinElement();
-    return createPortal(
-      <div>
-        {children}
-      </div>
-    , pinEl);
+    Object.assign(pinEl.style, {
+      height: "auto",
+      width: "auto",
+      fontSize: 0,
+    });
+    return createPortal(elementToRender, pinEl);
   }
 
   return null;
-}
+};
 
 Marker.defaultProps = {
   hideOffscreen: false,
   icon: (
-    <svg width="30" height="38" viewBox="0 0 30 38" fill="none" xmlns="http://www.w3.org/2000/svg">
-      <path d="M30 15.0882C30 23.4212 23.3333 30.7353 15 38C7.22222 31.2941 0 23.4212 0 15.0882C0 6.75523 6.71573 0 15 0C23.2843 0 30 6.75523 30 15.0882Z" fill="red" />
+    <svg
```

Comment:
Can you extract this to a named const outside of the component (aka the Marker const)?